api/assets/schemas.json

```diff
@@ -12491,6 +12491,340 @@
         },
         "$schema": "http://json-schema.org/draft-07/schema#"
     },
+    "ConnectionAuthCallbackSchema": {
```

Comment:
schemas.json is a generated file ( `npm run generate:schema` in `api` ), I hope that's what you did here, because the edits in this file seem a bit weird considering ConnectionAuthCallbackSchema only has like, 4 props in typescript

Reply:
> schemas.json is a generated file ( `npm run generate:schema` in `api` ), I hope that's what you did here, because the edits in this file seem a bit weird considering ConnectionAuthCallbackSchema only has like, 4 props in typescript

I'm not sure what happened, I did use the generate command.

Reply:
I regenerated the schema and it seems to be fixed now